Source/Scene/processPbrMetallicRoughness.js

```diff
@@ -578,9 +634,17 @@ define([
             fragmentShader += '    float VdotH = clamp(dot(v, h), 0.0, 1.0);\n';
 
             fragmentShader += '    vec3 f0 = vec3(0.04);\n';
-            fragmentShader += '    float alpha = roughness * roughness;\n';
-            fragmentShader += '    vec3 diffuseColor = baseColor * (1.0 - metalness) * (1.0 - f0);\n';
-            fragmentShader += '    vec3 specularColor = mix(f0, baseColor, metalness);\n';
+            if (useSpecGloss) {
+                fragmentShader += '    float alpha = pow((1.0 - glossiness), 2.0);\n';
+                fragmentShader += '    vec3 diffuseColor = diffuse.rgb * (1.0 - max(max(specular.r, specular.g), specular.b));\n';
+                fragmentShader += '    vec3 specularColor = specular;\n';
+                fragmentShader += '    float roughness = 1.0 - glossiness;\n';
```

Comment:
Do this above the `alpha` calculation on line 638 above, then you can avoid the `pow` and just have `float alpha = roughness * roughness`.